### PR TITLE
Updated discovery sensors module directory structure

### PR DESCRIPTION
**Pull request by laf**
This has annoyed me for long enough now!

The includes/discovery/ directory should contain the core discovery files with all of the code relevant to those in sub directories. Before this change all sensors files were scattered within here, these have now been moved to includes/discovery/sensors/

Everything needed should be updated ok.

### CODE REVIEW

**Comment by f0o** [comment]
trigger ci

**Comment by Rosiak** [comment]
:+1: 
No issues on dev machine.

File: includes/discovery/sensors.inc.php [hunk 1]
```diff
@@ -4,25 +4,25 @@
...
+require 'includes/discovery/sensors/temperatures.inc.php';
+require 'includes/discovery/sensors/humidity.inc.php';
+require 'includes/discovery/sensors/voltages.inc.php';
+require 'includes/discovery/sensors/frequencies.inc.php';
+require 'includes/discovery/sensors/current.inc.php';
+require 'includes/discovery/sensors/power.inc.php';
+require 'includes/discovery/sensors/fanspeeds.inc.php';
+require 'includes/discovery/sensors/charge.inc.php';
+require 'includes/discovery/sensors/load.inc.php';
 require 'includes/discovery/states.inc.php';
```
Comment (SaaldjorMike) [inline includes/discovery/sensors.inc.php]: Looks like this should also be updated to the new sensors folder.

**Comment by laf** [comment]
trigger ci

File: includes/discovery/sensors.inc.php (hunk 1 above)
Comment (laf) [inline includes/discovery/sensors.inc.php]: Well spotted. Updated.

**Pull request merged**